### PR TITLE
staking fixes

### PR DESCRIPTION
**Pull request by anubra266**
Staking page fixes from review

### CODE REVIEW

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
The changes are highlighted in the pictures below.
image

**Comment by anubra266** [comment]
@Simplypheyie96 look the size we have in figma.
CleanShot 2022-07-26 at 23 38 30@2x

File: src/components/icons/uniswap-ellipse.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        fill="#FF007A"
+      />
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M9.69177 10.0168C9.3993 10.0722 9.11497 10.2632 9.02692 10.4636C8.97323 10.5858 9.00371 10.8001 9.08415 10.8663C9.21407 10.9732 9.33972 11.0014 9.67997 10.9999C10.346 10.9971 10.9251 10.821 10.9925 10.6009C11.0477 10.4205 10.7933 10.1705 10.443 10.0608C10.2622 10.0041 9.87779 9.98166 9.69177 10.0168ZM10.4705 10.3921C10.5732 10.3021 10.5283 10.2049 10.3536 10.1392C10.0209 10.014 9.51773 10.1176 9.51773 10.3113C9.51773 10.4077 9.78008 10.5129 10.0206 10.5129C10.1807 10.5129 10.3997 10.4541 10.4705 10.3921Z"
+        fill="#FF007A"
+      />
```
Comment (anubra266) [inline src/components/icons/uniswap-ellipse.tsx:46]:
```suggestion
      />
      <circle cx="12" cy="12" r="12" fill="#FF007A" fillOpacity="0.2" />

**Comment by Simplypheyie96** [comment]
> ****

i don't know why that is showing you that. Please adjust as corrected
image
image

**Comment by Simplypheyie96** [comment]
@anubra266 please, always check figma well, i just realized you used the md breakpoint for the desktop to implement this. Do another check on figma with the correct labels

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
some fixes from the old review are yet to be attended to.
background color, the missing and the wrong icons

**Comment by anubra266** [comment]
CleanShot 2022-07-29 at 08 32 53@2x

@Simplypheyie96 The icon part is not clear. The user has to know when a ascending or descending sort is on, and when sort is off. Also, you didn't provide separate icons, the two are one svg. 
And if you tested the sort, you'll have seen that there are no icons by default, cause the data shouldn't be sorted by default. If you clicked any of the headers, it would be sorted and the icons would come up.

**Comment by Simplypheyie96** [comment]
> CleanShot 2022-07-29 at 08 32 53@2x
> 
> @Simplypheyie96 The icon part is not clear. The user has to know when a ascending or descending sort is on, and when sort is off. Also, you didn't provide separate icons, the two are one svg. And if you tested the sort, you'll have seen that there are no icons by default, cause the data shouldn't be sorted by default. If you clicked any of the headers, it would be sorted and the icons would come up.

i don't think that the ascending and descending sort is something on the design end. I can get you thesvg tho on figma beside the staking page. 
image

**Comment by anubra266** [comment]
@Simplypheyie96 I already figured out a way to get the separate svgs. Also I made some changes to show sort status with the icon colors. Check them out and lemme know if it's fine by you.

**Review by kesar — APPROVED** [review]

**Pull request merged**